### PR TITLE
[Website] Mega menu

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Updated mega menu to match figma design
- Changed dropdown menu to open on hover

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [WEB-156](https://airqoteam.atlassian.net/browse/WEB-156)

#### Screenshots
menu

### CODE REVIEW

File: website/frontend/config/urls.js [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-  process.env.REACT_APP_BASE_AIRQLOUDS_URL || process.env.REACT_NETMANAGER_BASE_URL,
-);
+const BASE_AIRQLOUDS_URL = stripTrailingSlash("https://staging-platform.airqo.net/api/v1/");
 
 export const AIRQLOUD_SUMMARY = `${BASE_AIRQLOUDS_URL}/devices/airqlouds/summary?tenant=airqo`;
 
-const BASE_NEWSLETTER_URL = stripTrailingSlash(
-  process.env.REACT_APP_BASE_NEWSLETTER_URL || process.env.REACT_NETMANAGER_BASE_URL,
-);
+const BASE_NEWSLETTER_URL = stripTrailingSlash("https://staging-platform.airqo.net/api/v1/");
```
Comment (FaithDaka) [inline website/frontend/config/urls.js:7]: Newsletter url routes to the platform?

**Review by FaithDaka — APPROVED** [review]

File: website/frontend/config/urls.js (hunk 1 above)
Comment (Codebmk) [reply website/frontend/config/urls.js:7]: @ja-odur should I leave these changes in the urls file or I should change it back?

Comment (ja-odur) [reply website/frontend/config/urls.js:7]: @Codebmk Don't check this in. This was a quick windows hack. We will get a better solution

**Review by FaithDaka — APPROVED** [review]

File: website/frontend/config/urls.js (hunk 1 above)
Comment (Codebmk) [reply website/frontend/config/urls.js]: @ja-odur I have removed them. Thanks.

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Review by ja-odur — APPROVED** [review]
@Codebmk In the near future we might want to centre/balance the position of the mega menu.

**Pull request merged**